android/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java

```diff
@@ -199,6 +213,110 @@ private boolean isAllowedUrl(Uri loadingUrl) {
         return !(bridge.getServerUrl() == null && !bridge.getAppAllowNavigationMask().matches(loadingUrl.getHost()));
     }
 
+    private Boolean isDomainExcludedFromSSL(Bridge bridge, URL url) {
```

Comment:
could we make the same method we have in `HttpRequestHandler` public and reuse it here instead of duplicating, or move it somewhere else were we can make it protected and then use it in both places?